examples/getting-started-session-based/03-serving-session-based-model-torch-backend.ipynb

```diff
@@ -63,18 +63,7 @@
    "execution_count": 2,
```

Comment:
<p>@oliver this number  <code>486</code>  does not match with the cardinality in the workflow.output_schema. are you using the different saved workflow by any chance? each time you rerun ETL you will get a different num of unique items since we gen data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/677/discussion/#comment-1177937304'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
let me re-run both notebooks again. I only commited the part of the first notebook that had changed

Reply:
<p>run all the notebooks now and should match up now with the schema</p>